### ISSUE TITLE
Get Non-recent Articles

### ISSUE THREAD

**Issue by Cohen-Ben** [codelucas/newspaper#342]
Is there any way to get articles that aren't classified as _recent_? Most of the articles I'm coming across are from the last day or so, but what if I want articles from before that? Is there any way to do so?

**Comment by juskaiser** [comment]
Agreed, this is something that I'm interested in. I would like to be able to look into archived articles.
